thirdparty/qtsingleapplication/src/qtlocalpeer.cpp

```diff
@@ -62,46 +62,30 @@ const char* QtLocalPeer::ack = "ack";
 QtLocalPeer::QtLocalPeer(QObject* parent, const QString &appId)
     : QObject(parent), id(appId)
 {
-    QString prefix = id;
     if (id.isEmpty()) {
-        id = QCoreApplication::applicationFilePath();
-#if defined(Q_OS_WIN)
-        id = id.toLower();
-#endif
-        prefix = id.section(QLatin1Char('/'), -1);
+        id = QCoreApplication::applicationName();
     }
+    QString prefix = id;
+#if defined( Q_OS_WIN )
+    prefix = prefix.toLower();
+#endif
+
     prefix.remove(QRegularExpression("[^a-zA-Z]"));
     prefix.truncate(6);
 
-    QByteArray idc = id.toUtf8();
+    QByteArray idc = QDir::home().dirName().toUtf8();
     quint16 idNum = qChecksum(idc.constData(), idc.size());
-    socketName = QLatin1String("qtsingleapp-") + prefix
+    socketName = QLatin1String("single-") + prefix
                  + QLatin1Char('-') + QString::number(idNum, 16);
 
-#if defined(Q_OS_WIN)
-    if (!pProcessIdToSessionId) {
-        QLibrary lib("kernel32");
-        pProcessIdToSessionId = (PProcessIdToSessionId)lib.resolve("ProcessIdToSessionId");
-    }
-    if (pProcessIdToSessionId) {
-        DWORD sessionId = 0;
-        pProcessIdToSessionId(GetCurrentProcessId(), &sessionId);
-        socketName += QLatin1Char('-') + QString::number(sessionId, 16);
-    }
-#else
-    socketName += QLatin1Char('-') + QString::number(::getuid(), 16);
-#endif
-
     server = new QLocalServer(this);
     QString lockName = QDir(QDir::tempPath()).absolutePath()
                        + QLatin1Char('/') + socketName
                        + QLatin1String("-gdlockfile");
     lockFile = new QLockFile(lockName);
     lockFile->setStaleLockTime(0);
     if ( !lockFile->tryLock( 0 ) ) {
-        if ( lockFile->error() != QLockFile::NoError && lockFile->error() != QLockFile::LockFailedError ) {
```

Comment:
The `QLockFile::LockFailedError` happens when `someone else already locked it".

It happens when starting a new instance.

We want this "error" to happen so that we know there is already an instance running.

So it should not print a warning, LockFailedError should not print a warning.

This part should not be changed

Reply:
when lock failed ,print the reason.  users who check the log file can determine at that time.   
It will give users a clear viewpoint and choice.

And consider it was only a log,  print  a little much more is ok.

Reply:
It will print when searching a word via command line when nothing wrong happens:

![image](https://user-images.githubusercontent.com/20123683/236609168-ecc9baf6-0dac-4946-9848-e873e90f096f.png)

Reply:
done

Reply:
And about the indentation .   the clang-format seems does not take effect in the first `if`.